python-backend/constants.py

```diff
@@ -5,8 +5,8 @@
 HOSTS = ['localhost', '127.0.0.1']
 PORTS = ['5173', '80']
 # how many lines of a series to print for debugging purposes
-DEBUG_LINES = 100
```

Comment:
I'm no longer debugging here so I reduced the number of logged debug lines for the time being (eventually will disable debug altogether)